src/config.ts

```diff
@@ -583,19 +583,24 @@ export function stripAndRedirectConfig(config: Config) {
  * For subtitle configs in config.title, keep them in config.title as header titles never have subtitles.
  */
 function redirectTitleConfig(config: Config) {
-  const {mark: m, subtitle} = extractTitleConfig(config.title);
-
-  const style: MarkConfig = {
-    ...config.style['group-title'],
-    ...m
-  };
-
-  // set config.style if it is not an empty object
-  if (keys(style).length > 0) {
-    config.style['group-title'] = style;
+  const {titleMarkConfig, subtitleMarkConfig, subtitle} = extractTitleConfig(config.title);
+
+  // set config.style if title/subtitleMarkConfig is not an empty object
+  if (keys(titleMarkConfig).length > 0) {
+    // TODO figure precedence
```

Comment:
Why not do this here? Maybe file an issue instead of just a todo in the code.

Reply:
forgot to remove.